libpromises/match_scope.h

```diff
@@ -26,6 +26,10 @@
 #define CFENGINE_MATCH_SCOPE_H
 
 #include <cf3.defs.h>
+#include <regex.h>
+
+/* Does not free rx! */
+bool PartialMatch(const pcre *rx, const char *teststring);
```

Comment:
I would prefer a function name that indicates this has to do with regexes, like `RegexPartialMatch()`.